packages/visual-editor/vitest.config.ts

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
   //@ts-expect-error ignore
   plugins: [tsconfigPaths()],
   test: {
+    globals: true,
```

Comment:
[note for reviewer] allows to use `describe`, `it`, etc.. without having to import them everyime from vitest